src/universal/modules/teamDashboard/components/ProjectColumn/ProjectColumnDropZone.js

```diff
@@ -1,23 +1,28 @@
 // @flow
 import type {Node} from 'react';
-import type {Project} from 'universal/types/project';
+import type {Project, ProjectID, Status} from 'universal/types/project';
 
 import React from 'react';
 import {DropTarget} from 'react-dnd';
 
 import withAtmosphere from 'universal/decorators/withAtmosphere/withAtmosphere';
 import sortOrderBetween from 'universal/dnd/sortOrderBetween';
-import withDragCache, {DragCache} from 'universal/dnd/withDragCache';
 import UpdateProjectMutation from 'universal/mutations/UpdateProjectMutation';
 import {PROJECT} from 'universal/utils/constants';
 
 type Props = {
   connectDropTarget: (node: Node) => Node,
   area: string,
   atmosphere: Object, // TODO: atmosphere needs a type definition
-  dragCache: DragCache,
+  getProjectById: (ProjectID) => Project,
   lastProject: ?Project, // the last project in a column; may be undefined if the column is empty
-  status: string
+  status: Status
+};
+
+type UpdateProjectMutationArgs = {
```

Comment:
oh man i want to so badly import this from our graphql schema!

Reply:
Yeah, same! Happy to do that the next time around :)